packages/itwin/measure-tools/package.json

```diff
@@ -47,41 +47,42 @@
     "!lib/**/test/**/*"
   ],
   "peerDependencies": {
-    "@itwin/appui-abstract": "^3.0.0",
-    "@itwin/appui-layout-react": "^3.0.0",
-    "@itwin/appui-react": "^3.0.0",
-    "@itwin/components-react": "^3.0.0",
-    "@itwin/core-bentley": "^3.0.0",
-    "@itwin/core-common": "^3.0.0",
-    "@itwin/core-frontend": "^3.0.0",
-    "@itwin/core-geometry": "^3.0.0",
-    "@itwin/core-react": "^3.0.0",
-    "@itwin/core-telemetry": "^3.0.0",
+    "@itwin/appui-abstract": "^3.7.0",
+    "@itwin/appui-layout-react": "^4.0.0.dev-13",
+    "@itwin/appui-react": "^4.0.0-dev.13",
+    "@itwin/components-react": "^4.0.0-dev.13",
+    "@itwin/core-bentley": "^3.7.0",
+    "@itwin/core-common": "^3.7.0",
+    "@itwin/core-frontend": "^3.7.0",
+    "@itwin/core-geometry": "^3.7.0",
+    "@itwin/core-react": "^4.0.0-dev.13",
+    "@itwin/core-telemetry": "^3.7.0",
```

Comment:
i think this might be ok as a direct dep
so remove as peer+dev, and make it direct

Reply:
moved